pkg/tests/conftest.py

```diff
@@ -33,7 +33,12 @@ def version():
             artifact.name,
         )
         if _version:
-            _version = _version.groups()[0].replace("_", "-").replace("~", "")
+            _version = (
+                _version.groups()[0]
+                .replace("_", "-")
+                .replace("~", "")
+                .replace("-py3-x86-64", "")
```

Comment:
Have a fix with changes this in https://github.com/saltstack/salt/pull/63565, better accommodates tarballs and addresses the regex issue.
Btw. this won't work for Apple Mx which is coming